scripts/ci

```diff
@@ -5,10 +5,7 @@ SCRIPT_DIR=$(dirname $0)
 pushd $SCRIPT_DIR
 
 ./download
-if [ -z "$SKIP_VALIDATE" ]; then
```

Comment:
Why was this removed?

Reply:
because it's down below in the validate script now instead of in the called script. its to make the go generate stuff work in the gh actions who were skipping validation but they still need the data/manifests stuff

Reply:
Does this build if the branch is dirty?

Reply:
should, why do you suspect it wont?